crates/matrix-sdk/src/widget/machine/mod.rs

```diff
@@ -321,7 +338,7 @@ impl WidgetMachine {
             }),
         };
 
-        let action = if capabilities.send.iter().any(|filter| filter.matches(&filter_in)) {
+        if capabilities.send.iter().any(|filter| filter.matches(&filter_in)) {
```

Comment:
Please early return here too for consistency, since you did the same above in `process_read_event_request`.

Reply:
Makes sense! Updated.